client/utils/auth.py

```diff
@@ -7,6 +6,9 @@
 import sys
 import time
 import webbrowser
+from urllib.request import urlopen
+import json
+from .html import auth_html, partial_course_html, partial_nocourse_html, red_css
```

Comment:
I originally tried putting all HTML templates in a directory called "templates," but it wasn't included in the binary, so I created a new python file `html.py` with all the html contents. The prepending dot is the current directory. Should I add a comment, or is should I rename it "templates.py".?